apps/main/src/dex/components/PageIntegrations/index.tsx

```diff
@@ -144,14 +144,16 @@ const IntegrationsComp = ({
               integrationsAppNetworks={
                 !rChainId && (
                   <Box margin="0.25rem 0 0 0">
-                    {Object.keys(app.networks).map((networkId) => {
-                      if (networkId in networksIdMapper) {
```

Comment:
You can delete line 40 in the new file now, as networksIdMapper is now unused

`const networksIdMapper = useStore((state) => state.networks.networksIdMapper)`